scripts/zones/Mine_Shaft_2716/mobs/Bugboy.lua

```diff
@@ -0,0 +1,71 @@
+-----------------------------------
+-- Area: Mine Shaft 2716
+-- Mob: Bugboy
+-- ENM : Bionic Bug
+-----------------------------------
+mixins = { require('scripts/mixins/job_special') }
+-----------------------------------
+---@type TMobEntity
+local entity = {}
+
+entity.onMobSpawn = function(mob)
+    mob:setMobMod(xi.mobMod.WEAPON_BONUS, 20)
+end
+
+entity.onMobFight = function(mob, target)
+    if xi.combat.behavior.isEntityBusy(mob) then
+        return
+    end
+
+    -- Use Mighty Strikes at 75%, 50%, and 25% HP.
+    local hPP = mob:getHPP()
+    local mightyStrikes = mob:getLocalVar('mightyStrikes')
```

Comment:
sir, they do not charge us on a per-character basis. We arent DSP.

You could (and maybe even should) have used the var names and indentation I provided.